orttraining/orttraining/core/graph/allreduce_optimizer_graph_builder.cc

```diff
@@ -174,24 +142,20 @@ Status AllreduceOptimizerGraphBuilder::BuildInternal(
     return graph.GenerateNodeArgName(base_name);
   };
 
-  const bool overlap_compute_allreduce = !opt_graph_config_.use_nccl;
   const int64_t horovod_reduce_op = opt_graph_config_.horovod_reduce_op;
 
   // add gradient scaling
-  ArgDef fused_gradient_argdef;
+  std::vector<ArgDef> fused_gradient_argdef;
```

Comment:
Is this still used?  Or can it be removed?

Reply:
I have converted it to a vector, so instead of fusion now we have individual tensors 

---
In reply to: [505116558](https://github.com/microsoft/onnxruntime/pull/4443#discussion_r505116558) [](ancestors = 505116558)